README.md

```diff
@@ -112,6 +113,17 @@ end
 grant_on ['users#create', 'users#update'], badge: 'autobiographer', temporary: true do |user|
   user.name? && user.email?
 end
+
+grant_on 'users#update', badge: 'updated_my_info' do |user, current_user:|
+  user == current_user
+end
```

Comment:
This example can get confusing as we speak of `user` in two different contexts. Let's do:

```ruby
grant_on 'addresses#update', badge: 'address_updated' do |address, current_user:|
  current_user == address.user
end
```